### DISCUSSION TITLE
No Galaxy Brain acheivement even after answered discussion

### DISCUSSION THREAD

**Discussion by yashash-pugalia** [community/community#18838] [Other Feature Feedback, Questions, & Ideas] [answered]
I didn't get the galaxy brain acheivement even though i have answered discussions.

**Comment by aenq** [comment, marked as answer]
Because your answer didn't marked as answer by the owner of the question. Just keep answering discussion.

**Reply by yashash-pugalia** [reply]
if i mark my answer on a discussion question by someone else.(repo not owned by me, though i am a collaborator) it doesn't work
